core/src/test/scala/integration/kafka/api/AdminClientTest.scala

```diff
@@ -57,7 +64,14 @@ class AdminClientTest extends IntegrationTestHarness with Logging {
   @Before
   override def setUp() {
     super.setUp
+
+    val nClient = EasyMock.mock(classOf[ConsumerNetworkClient]);
```

Comment:
It looks like all of this setup is only needed for the single test below. Maybe we should just move it into that test case?